content/docs/user-guide/components/reference/physx/collider.md

```diff
@@ -1,258 +1,148 @@
-Triangle meshes are not supported as triggers.
-
-**Simulated**
-When enabled, this shape collider will be part of the physics simulation.
-
-**In Scene Queries**
-When enabled, this shape collider can be queried for raycasts, shapecasts and overlap.
-
-**Offset**
-Local offset position of the collider, relative to the entity.
-
-**Rotation**
-Local rotation of the collider about the **Offset** of the PhysX collider component.
-
-**Library (Physics Materials)**
-The physics material library of the project.
-
-**Slots (Physics Materials)**
-Choose a physics material for each material of this collider. A collider can have multiple materials assigned. For more information, see [Physics materials](/docs/user-guide/interactivity/physics/nvidia-physx/materials/).
-
-**Tag**
-Set a tag for this collider. Tags can be used to quickly identify components in script or code.
+## Provider
 
-**Rest offset**
-PhysX bodies come to rest separated by the sum of their rest offset values. The **Rest offset** value must be less than the **Contact offset** value. Valid values rage from **-Infinity** to **50**.
+[PhysX Gem](/docs/user-guide/gems/reference/physics/nvidia/physx/)
 
-**Contact offset**
-PhysX bodies generate contacts when they are within the sum of their contact offset values. The **Contact offset** value must be greater than the **Rest offset** value. Valid values rage from **0** to **50**.
+## Properties 
 
-**Shape**
-Shape of the collider. A collider can be a primitive shape or a physics asset.
-To use a primitive shape, choose **Box**, **Sphere**, or **Capsule**. For more information, see [Primitive colliders](#primitive-colliders).
-To use a physics asset, choose **Physics Asset**. For more information, see [Physics asset colliders](#physics-asset-colliders).
+![PhysX Collider component interface.](/images/user-guide/components/reference/physx/physx-collider-ui-01.png)
 
-**Physics Asset shape**
+### Base properties
 
-![PhysX Collider component interface, Physics Asset.](/images/user-guide/component/physx/physx/ui-physx-collider-A-0.png)
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Collision Layer** | Assigns the collider to a collision layer. Collision layers can be used to restrict physical interactions between PhysX objects. | Any collision layer defined in the project's [collision layers](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).  | `Default` |
+| **Collides With** | Assigns the collider to a collision group. Collision groups contain the collision layers that this collider can collide with. | Any collision group defined in the project's [collision groups](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups/). | `All` |
```

Comment:
Changing these to match the shape collider page, and because I think they are the GUI labels and will ultimately be the topic titles.

```suggestion
| **Collision Layer** | Assigns the collider to a collision layer. Collision layers can be used to restrict physical interactions between PhysX objects. | Any collision layer defined in the project's [Collision Layers](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-layers/).  | `Default` |
| **Collides With** | Assigns the collider to a collision group. Collision groups contain the collision layers that this collider can collide with. | Any collision group defined in the project's [Collision Groups](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-collision-groups/). | `All` |
```